### PR TITLE
Add Stacked Icons to Project Cards

### PR DESCRIPTION
**Pull request by ayesha-omarali**
- Added PlatformList component (stacked icons) to project cards
- Updated tests

image
https://getsentry.atlassian.net/browse/APP-370

### CODE REVIEW

File: tests/js/spec/views/organizationDashboard/projectCard.spec.jsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ describe('ProjectCard', function() {
   });
 
   it('renders with platform', function() {
-    expect(wrapper.find('Platformicon[platform="javascript"]')).toHaveLength(1);
+    expect(wrapper.find('PlatformList')).toHaveLength(1);
```
Comment (billyvg) [inline tests/js/spec/views/organizationDashboard/projectCard.spec.jsx]: We should make sure this renders the javascript icon

**Review by ckj — APPROVED** [review]
Visuals look 👍

File: tests/js/spec/views/organizationDashboard/projectCard.spec.jsx (hunk 1 above)
Comment (ayesha-omarali) [reply tests/js/spec/views/organizationDashboard/projectCard.spec.jsx]: Done! thank you 😄

File: src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx [hunk 2]
```diff
@@ -62,13 +62,19 @@ class ProjectCard extends React.Component {
       ? t('Remove from bookmarks')
       : t('Add to bookmarks');
 
+    let platforms;
+    if (typeof project.platforms !== 'undefined') {
+      platforms = project && project.platforms;
+    } else {
+      platforms = [];
+    }
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx]: no need to change but couldn't this be `const platforms = project && project.platforms || []`?

**Comment by billyvg** [comment]
Yeah I looked the other day, we should change it to display 2 icons by default:
image

**Comment by benvinegar** [comment]
> Yeah I looked the other day, we should change it to display 2 icons by default:

I feel like I'm missing something. Why default to 2 when 90% of people have 1? What does "display 2 icons by default" mean?

File: src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx [hunk 3]
```diff
@@ -62,13 +62,19 @@ class ProjectCard extends React.Component {
...
     return (
       <ProjectCardWrapper data-test-id={slug} width={['100%', '50%', '33%', '25%']}>
         {stats ? (
           <StyledProjectCard>
             <Flex justify="space-between" align="center">
               <Box ml={2}>
-                <StyledPlatformicon size="24" platform={project.platform || 'generic'} />
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx]: just fyi, the icon sizes will be different here

**Comment by billyvg** [comment]
10% feels like a significant enough population that we should have the option to show up to 2 platform icons (instead of the current 3).

**Pull request merged**